benchmarks/run_import.py

```
@@ -0,0 +1,130 @@
+    elapsed = (time.perf_counter() - t0) * 1000
+    print(f"ERR {elapsed:.2f} {e}")
+"""
+
+
+def measure_subprocess(script: str, rounds: int = 5) -> dict:
+    """Run a Python snippet in a fresh process and collect timings."""
+    timings = []
+    errors = []
+
+    for _ in range(rounds):
+        result = subprocess.run(
```

Ruff (0.15.5)

[error] `subprocess` call: check for execution of untrusted input

(S603)